.gitignore

```diff
@@ -7,3 +7,6 @@ manual/
 dist/
 __pycache__
 .php_cs.cache
+psysh.phar
+psysh-compat.phar
+.idea/
```

Comment:
Please add this one to your global gitignore rather than leaving it as a little present in every open source project you contribute to :)

Reply:
Good point - done!